packages/blade/src/utils/cssBezierToMotionFn.ts

```diff
@@ -0,0 +1,24 @@
+const cssBezierToMotionFn = <
```

Comment:
Bit misleading name, it says "bezier to motion fn" but returns an array. :p 

CSSBezierToArray